addons/hr_contract/data/hr_contract_demo.xml

```diff
@@ -16,6 +16,96 @@
         </record>
     </data>
 
+    <record id="resource_calendar_std_20h" model="resource.calendar">
+        <field name="name">Standard 20 hours/week</field>
+        <field name="company_id" ref="base.main_company"/>
+        <field name="hours_per_day">4.0</field>
+        <field name="attendance_ids"
+            eval="[Command.clear(),
+                Command.create({'name': 'Monday Morning', 'dayofweek': '0', 'hour_from': 8, 'hour_to': 12, 'day_period': 'morning'}),
```

Comment:
Missing lunch periods.

Reply:
Please use magic number (0, 0, dict)

Reply:
I thought we are minimizing use of magic numbers :thinking: 
(As I got some comments about using `Command` instead magic numbers in previous PRs)

Reply:
I think this is quite subjective, according to the reviewer sensibility. You can keep it that way :)